packages/theming/atlas/tests/e2e/allScreens.spec.ts

```diff
@@ -0,0 +1,44 @@
+const fs = require("fs");
+const { join, resolve } = require("path");
+const cwd = process.cwd();
+const xmlParser = require("fast-xml-parser");
+
+// For every module mx exports a folder, and in that folder we have bunch of xml's
+// which might contains URL field
+const individualPageFolders = join(cwd, "tests/testProject/deployment/web/pages/en_US");
+
+// Make an async function that gets executed immediately
+function* getFiles(dir) {
+    const dirents = fs.readdirSync(dir, { withFileTypes: true });
+    for (const dirent of dirents) {
+        const res = resolve(dir, dirent.name);
+        if (dirent.isDirectory()) {
+            yield* getFiles(res);
+        } else {
+            yield res;
+        }
+    }
+}
+
+describe("Screenshots of the pages for", () => {
+    beforeAll(() => {
+        browser.url("/");
+    });
+
+    for (const f of getFiles(individualPageFolders)) {
```

Comment:
Nit: `f` is a non-descriptive variable. `file` would be better imo.

Reply:
IIUC, `getFiles` actually returns a list of file paths. `getFilePaths` would be a clearer name for the function, leading to `filePath` instead of `f`.

Reply:
Good idea ✔️